annif/backend/backend.py

```diff
@@ -80,6 +81,8 @@ def suggest(self, text, params=None):
         represented as a list of SubjectSuggestion objects."""
         beparams = self._get_backend_params(params)
         self.initialize()
+        if int(beparams['input_limit']) != 0:
```

Comment:
What happens if `input_limit` is set to zero in the configuration file? The string `0` is different from 0, so the `if` clause is true. Will the input then be truncated to zero length?

Reply:
That's no problem, the `input_limit` is converted to `int`, so even if it is string from the configuration file, the condition works as expected.

Reply:
So no truncation is performed.

Reply:
You're right